federation/src/test/scala/caliban/federation/tracing/FederationTracingSpec.scala

```diff
@@ -33,20 +33,22 @@ object FederationTracingSpec extends ZIOSpecDefault {
 
   case class Queries(me: ZQuery[Any, Nothing, User])
 
-  def api(excludePureFields: Boolean) = graphQL(
-    RootResolver(
-      Queries(
-        me = ZQuery.succeed(
-          User(
-            "abc123",
-            ZIO.sleep(100.millis) as Name("my_first", Some("my_last")),
-            age = 42,
-            parents = ZIO.succeed(List(Parent("my_parent")))
+  def api(excludePureFields: Boolean) = ZIO.clock.map { implicit clock =>
```

Comment:
Yeah definitely not sold on this 😄  I don't think running that effect once per query is significant at all 🤔

Reply:
Ok I'll change these ones back